infra/scripts/build-run-docker.sc

```diff
@@ -6,9 +6,9 @@ import $file.common.splash
 import $file.common.vars
 import $file.docker.buildRunDocker
 
-import common.vars._
```

Comment:
eee.... does it mean we have different ammonite version or both import versions work?

Reply:
Yep, confirmed it works both ways. Yours is better.